README.md

```diff
@@ -98,8 +98,7 @@ WebUIs:
 Model galleries
 - https://github.com/go-skynet/model-gallery
   
-Auto Docker / Model setup
-- https://io.midori-ai.xyz/howtos/easy-localai-installer/
+LocalAI Manager (By Midori AI)
```

Comment:
I think this should be a category. Maybe we can have a UI/Management catch-all

Reply:
That sounds lovely. What do you want it called